### PR TITLE
Update connect-sharepoint-online.md

### PR DESCRIPTION
**Pull request by Michal-Ziemba**
Added a note about incompatibility to PowerShell ver 7 and how to mitigate it.

### CODE REVIEW

File: sharepoint/docs-conceptual/sharepoint-online/connect-sharepoint-online.md [hunk 1]
```diff
@@ -36,6 +36,11 @@ To open the SharePoint Online Management Shell command prompt, from the **Start*
 
 > [!VIDEO https://www.youtube.com/embed/TMzHAWEQjlk]
 
+> [!NOTE]
+> The SharePoint Online Management Shell primarily targets PowerShell 5.x, which means it might not be compatible with PowerShell 7 due to the significant changes in the underlying architecture between the two versions. To use it in PowerShell 7, consider importing the module using the Windows PowerShell Compatibility feature:
+>   ```powershell
+>   Import-Module Microsoft.Online.SharePoint.PowerShell -UseWindowsPowerShell
+
```
Comment (JohanFreelancer9) [inline sharepoint/docs-conceptual/sharepoint-online/connect-sharepoint-online.md:43]:
```suggestion
>   ```

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Michal-Ziemba, kindly commit the suggested change. Many thanks.

**Comment by get-itips** [comment]
Maybe @salarson should review this one too @yogkumgit

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@salarson please review. Thanks

**Review by salarson — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Michal-Ziemba Closing this PR, as this is not approved. Thanks for taking out some time and creating this PR. Appreciate and encourage you to do the same in future also.

**Pull request closed without merging**